code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -41,6 +41,23 @@
 	gold_core_spawnable = 0
 
 /mob/living/simple_animal/pet/cat/Life()
+	if(!stat && !buckled)
+		if(prob(1))
+			emote("me", 1, pick("stretches out for a belly rub.", "wags its tail."))

```

Comment:
cats don't really wag....swish maybe, but not wag.

Also, cats and belly rugs are two combinations that notoriously do not mix.